hyperspy/_components/rc.py

```diff
@@ -16,24 +16,38 @@
 # You should have received a copy of the GNU General Public License
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
-import numpy as np
+from hyperspy._components.expression import Expression
 
-from hyperspy.component import Component
-
-
-class RC(Component):
+class RC(Expression):
 
     """
-    """
+    RC function component (based on the time-domain capacitor voltage response of an RC-circuit)
+    
+    .. math::
 
-    def __init__(self, V=1, V0=0, tau=1.):
-        Component.__init__(self, ('Vmax', 'V0', 'tau'))
-        self.Vmax.value, self.V0.value, self.tau.value = V, V0, tau
+        f(x) = V0 + Vmax * (1 - exp(-x / \tau))
```

Comment:
Use latex to display nicely the formula (see for example https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/hyperspy/_components/lorentzian.py#L30)?

Add a parameter/attribute mapping table as in https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/hyperspy/_components/lorentzian.py#L32 to make it easier to read?

Reply:
done. see #2148 for discussion on table format